### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by Het-Shah**
This gives correct docs. Part of #104

### CODE REVIEW

**Comment by Het-Shah** [comment]
I'll resolve conflicts later.

**Comment by Het-Shah** [comment]
image

**Comment by Sharad24** [comment]
gawd

**Comment by Het-Shah** [comment]
LOL.

File: genrl/deep/agents/dqn/dqn.py [hunk 1]
```diff
@@ -32,29 +32,45 @@
...
-    :param tensorboard_log: (str) the log location for tensorboard
-        (if None, no logging)
-    :param seed : (int) seed for torch and gym
-    :param render : (boolean) if environment is to be rendered
-    :param device : (str) device to use for tensor operations; 'cpu' for cpu
-        and 'cuda' for gpu
+    Paper (DQN) https://arxiv.org/pdf/1312.5602.pdf
+    Paper (Double DQN) https://arxiv.org/abs/1509.06461
+
+    :param network_type: The deep neural network layer types ['mlp']
```
Comment (Zeus3101) [inline genrl/deep/agents/dqn/dqn.py:38]: You can add `'cnn'` also as it's been merged

Comment (Het-Shah) [reply genrl/deep/agents/dqn/dqn.py:38]: yea sorry, i'll add this

File: genrl/deep/agents/dqn/dqn.py [hunk 2]
```diff
@@ -144,7 +160,13 @@ def __init__(
 
         self.create_model()
 
-    def create_model(self):
+    def create_model(self, network_type):
+        '''
```
Comment (Zeus3101) [inline genrl/deep/agents/dqn/dqn.py:164]: Network type is a class attribute now, no need to pass it as an argument

**Comment by Het-Shah** [comment]
Merge this now

**Comment by Sharad24** [comment]
Ok sir

**Pull request merged**